.circleci/verbatim-sources/linux-build-defaults.yml

```diff
@@ -22,6 +22,9 @@ pytorch_linux_build_defaults: &pytorch_linux_build_defaults
         # Pull Docker image and run build
         echo "DOCKER_IMAGE: "${DOCKER_IMAGE}
         docker pull ${DOCKER_IMAGE} >/dev/null
+
+        DOCKER_IMAGE=${DOCKER_IMAGE} DOCKER_IMAGE_TYPE=pytorch/base source /home/circleci/project/.circleci/scripts/upload_docker_image_to_s3.sh
```

Comment:
Doesn't this imply you are uploading the *base* docker image to S3 on every single build? That doesn't sound too good to me